src/meta/Dimensions.ts

```diff
@@ -23,7 +25,7 @@ export interface DimensionResults {
 	scroll: TopLeft & Size;
 }
 
-const defaultDimensions = {
+export const defaultDimensions = {
```

Comment:
thinking about this, doesn't this mean that something external could mess with the defaults?